public/app/features/plugins/admin/hooks/usePluginConfig.tsx

```diff
@@ -11,7 +11,11 @@ export const usePluginConfig = (plugin?: CatalogPlugin) => {
       return null;
     }
 
-    const isPluginInstalled = config.pluginAdminExternalManageEnabled ? plugin.isFullyInstalled : plugin.isInstalled;
+    // On Cloud, check both isFullyInstalled (for multi-instance setup) and isInstalled (fallback for single instance)
+    // This ensures tabs show even if instance data hasn't fully loaded
+    const isPluginInstalled = config.pluginAdminExternalManageEnabled
```

Comment:
I initially approved this but now I am worried that this might actually introduce an erratic behaviour in cloud since isInstalled might be true when isFullyInstalled is false in cloud.

The best person to verify this is @oshirohugo who introduced these flags and checks.